components/tests/ui/robot_setup.sh

```diff
@@ -57,6 +68,13 @@ do
   bin/omero import -d $delDs $TINY_IMAGE_NAME --debug ERROR
 done
 
+# Import Plate
+bin/omero import $PLATE_NAME --debug ERROR > plate_import.log 2>&1
+plateid=$(sed -n -e 's/^Plate://p' plate_import.log)
+# Use populate_metadata to upload and attach bulk annotation csv
+PYTHONPATH=$PYTHONPATH:lib/python
+python lib/python/omero/util/populate_metadata.py -k $key Plate:$plateid $BULK_ANNOTATION_CSV
```

Comment:
If this is merged in to the Metadata branch instead of develop, we'll still have the same problem later when we try to merge Metadata back into develop. Does https://github.com/joshmoore/openmicroscopy/commit/f12a950990d0866bf0ce9393710a25377bcdc567 help? Can we get that into develop?